cmd/internal/core/register-switch.go

```diff
@@ -20,12 +21,34 @@ func (c *Core) RegisterSwitch() error {
 		managementUser string
 	)
 
+	err = retry.Do(
+		func() error {
+			initialized, err := c.nos.IsInitialized()
+			if err != nil {
+				return err
+			}
+			if initialized {
+				return nil
+			}
+			return fmt.Errorf("switch is not yet initialized, retrying")
+		},
+		retry.Attempts(20),
+		retry.Delay(1*time.Second),
```

Comment:
My gut tells me that twenty seconds may be too low. Maybe one minute?

Reply:
`DefaultDelayType     = CombineDelay(BackOffDelay, RandomDelay)`
so the delay basically doubles on every try ... with 20 attempts this would result at the end to a couple of weeks? :-)